src/pages/VolunteersFind.page

```diff
@@ -41,16 +41,8 @@
     </style>
     <script type="text/javascript">
     (function($) {
-        //impossible to SLDS style visualforce multiselect picklists, instead we'll use a standard select
-        function fixMultiPicklists() {
-            $('.multiSelectPicklistTable').hide();
-            $('.multipicklistSLDSFixup').attr("size", 4);
```

Comment:
any way you could get this setting of size back to 4 in the find page.  It makes the top section stop using up so much darn space under SLDS.